src/layout/vertical-nav/VerticalNavGroup.tsx

```diff
@@ -39,8 +42,9 @@ interface VerticalNavGroupState {
 export class VerticalNavGroup extends React.PureComponent<VerticalNavGroupProps, VerticalNavGroupState> {
     constructor(props: VerticalNavGroupProps) {
         super(props);
+        const {isExpanded} = this.props;
         this.state = {
-            groupIsExpanded: false,
+            groupIsExpanded: isExpanded || false,
```

Comment:
It should be
```suggestion
            groupIsExpanded: !!props.isExpanded,
```
also please remove the line 45.